EnoDatabase/EnoDatabase.AttackInfo.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace EnoDatabase
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using EnoCore;
+    using EnoCore.AttackInfo;
+    using EnoCore.Configuration;
+    using EnoCore.Models;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Logging;
+
+    public partial class EnoDatabase : IEnoDatabase
+    {
+        public async Task<AttackInfo> GetAttackInfo(long roundId, Configuration config)
+        {
+            var teamNames = await this.context.Teams
+                .AsNoTracking()
+                .Select(t => new { t.Id, t.Address })
+                .ToDictionaryAsync(t => t.Id, t => t.Address);
+            var availableTeams = await this.context.RoundTeamServiceStatus
+                .Where(rtss => rtss.GameRoundId == roundId)
+                .GroupBy(rtss => rtss.TeamId)
+                .Select(g => new { g.Key, BestResult = g.Min(rtss => rtss.Status) })
+                .Where(ts => ts.BestResult < ServiceStatus.OFFLINE)
+                .Select(ts => ts.Key)
+                .OrderBy(ts => ts)
+                .ToArrayAsync();
+            var availableTeamNames = availableTeams.Select(id => teamNames[id] ?? id.ToString()).ToArray();
+
+            var serviceNames = await this.context.Services
+                .AsNoTracking()
+                .Select(s => new { s.Id, s.Name })
+                .ToDictionaryAsync(s => s.Id, s => s.Name);
+
+            var relevantTasks = this.context.CheckerTasks
+                .AsNoTracking()
+                .Where(ct => ct.Method == CheckerTaskMethod.putflag)
+                .Where(ct => ct.CurrentRoundId <= roundId)
+                .Where(ct => ct.CurrentRoundId > roundId - config.FlagValidityInRounds)
+                .Where(ct => ct.CheckerResult == CheckerResult.OK)
```

Comment:
Is this needed/beneficial?

Reply:
After having a chat with @ldruschk, while it's currently only emitted for OK from the Python lib, it's indeed not specified that that needs to be the case and in fact probably makes sense to have on Mumble in certain situations, so I'll remove the check.

Reply:
Yep, we should remove this